openapi.yaml

```diff
@@ -323,10 +323,14 @@ info:
     ```
     A process node MUST always contain key-value-pairs named `process_id` and `arguments`. It MAY contain a `description`.
 
-    One of the nodes in a map of processes (the final one) MUST have the `result` flag set to `true`, all the other nodes can omit it as the default value is `false`. Having such a node is important as multiple end nodes are possible, but in most use cases it is important to exactly specify the return value to be used by other processes. Each child process graph must also specify a result node similar to the "main" process graph.
-
     `process_id` MUST be a valid process ID in the `namespace` given. Clients SHOULD warn the user if a user-defined process is added with the same identifier as one of the predefined process.
 
+    In the following, "end node" (also known as "leaf node") defines a node that is not referenced by any other node in the same process graph. The "root process graph" is the outermost process graph, which is not part of any other process graph.
```

Comment:
> "end node" (also known as "leaf node")

bike-shedding here: if I would have to pick, I'd prefer "leaf node" as main concept, as it feels more self-explanatory than "end node"

Reply:
I've just tested with two non-openEO persons and they both seemed to better understand end node. So I'm not sure what's actually more self-explanatory and would stick with it for now.